templates/resources/offset.md.tmpl

```diff
@@ -0,0 +1,35 @@
+---
+page_title: "{{.Name}} {{.Type}} - {{.ProviderName}}"
+subcategory: ""
+description: |-
+{{ .Description | plainmarkdown | trimspace | prefixlines "  " }}
+---
+
+# {{.Name}} ({{.Type}})
+
+{{ .Description | trimspace }}
+
+-> Further manipulation of incoming or outgoing values can be accomplished with the [`formatdate()` function](https://www.terraform.io/docs/configuration/functions/formatdate.html) and the [`timeadd()` function](https://www.terraform.io/docs/configuration/functions/timeadd.html).
+
+## Example Usage
+
+### Basic Usage
+
+{{ tffile "examples/resources/time_offset/resource.tf" }}
+
+### Triggers Usage
+
+{{ tffile "examples/resources/time_offset/resource_triggers.tf" }}
+
+{{ .SchemaMarkdown | trimspace }}
+~> **NOTE:** At least one of the `offset_` arguments must be configured.
+
+## Import
+
+This resource can be imported using the base UTC RFC3339 timestamp and offset years, months, days, hours, minutes, and seconds, separated by commas (`,`), e.g.
+
+```console
+$ terraform import time_offset.example 2020-02-12T06:36:13Z,0,0,7,0,0,0
+```
```

Comment:
This might be a bit "nitpicky", but here we should probably use the `.HasImport` and `.ImportFile` template data. Of course the result would be the same, but it would make us show _how it is done_. Let me find an example.

Reply:
So, something like this:

```gotmpl
{{ if .HasImport -}}
## Import

Import is supported using the following syntax:

{{codefile "shell" .ImportFile }}
{{- end }}
```

And the import file is expected to be located at: `examples/resources/<resource name>/import.sh`

Reply:
I'm looking at the other templates, and I feel like while valid to use the features of `tfplugindocs`, you are putting together such good template and import examples that is kinda superfluous.

So, entirely up to you. What matters is that the doc is good, and it's going to be 👍